scheduleit/config/hash.php

```diff
@@ -0,0 +1,41 @@
+<?php
```

Comment:
The scheduleit database.php is loading the same hash.php in the old config folder, so you don't need to add an extra one here.